include/tesseract/publictypes.h

```diff
@@ -285,7 +285,7 @@ enum OcrEngineMode {
 enum class ThresholdMethod {
```

Comment:
@amitdo, why is `ThresholdMethod` declared in the public API? It looks like it is unused there, so maybe we should remove it for release 5.0.0.

Reply:
I don't remember right now why I put it there.You can move it elsewhere.